src/hmrc-content-guide/forms-detailed-guide/index.njk

```diff
@@ -5,56 +5,99 @@ layout: twoColumnPage.njk
 ---
 {% from "_codeSnippet.njk" import codeSnippet %}
 
-<h2 class="govuk-heading-l">All forms</h2>
+<h2 class="govuk-heading-l">Document type</h2>
 
 <p class="govuk-body">
-The following forms will be accessible:
+Detailed guide
 </p>
 
-<ul class="govuk-list govuk-list--bullet">
-<li>a gForm (<a href="https://tax.service.gov.uk" class="govuk-link">https://tax.service.gov.uk</a>)</li>
-<li>in Open Document Format</li>
-</ul>
+<h2 class="govuk-heading-l">When to use</h2>
 
 <p class="govuk-body">
-For all other forms, you should include the following accessibility statement after the form:
+Before you use this pattern, you should <a href="https://design.tax.service.gov.uk/hmrc-content-guide/forms-check/" class="govuk-link">check which document type to use</a>, to decide
+if you should use a publication instead.
 </p>
 
-  {{
+<h2 class="govuk-heading-l">How it works</h2>
+
+<h3 class="govuk-heading-m">G-Forms</h3>
+
+<p class="govuk-body">
+G-Forms are hosted on <a href="https://tax.service.gov.uk" class="govuk-link">https://tax.service.gov.uk</a>. Depending on the design of the
+detailed guide, use either an inline link or a <a href="https://design.tax.service.gov.uk/hmrc-content-guide/green-button/" class="govuk-link">green button</a>.
+</p>
+
+<p class="govuk-body">
+These forms will not require an accessibility statement.
+</p>
+
+<h3 class="govuk-heading-m">Open Document Formats</h3>
+
+<p class="govuk-body">
+If the form is in Open Document Format it will not require an accessibility statement.
+</p>
+
+<p class="govuk-body">
+You must [use the publication box !@n markdown in
+Whitehall](<a href="https://www.gov.uk/guidance/how-to-publish-on-gov-
```

Comment:
Suggest this is just...

You must use the [publication box](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown#publication-box) !@n markdown in Whitehall.